models/demos/yolov4/ttnn/common.py

```diff
@@ -74,21 +74,20 @@ def __call__(self, device, input_tensor):
         conv_config = ttnn.Conv2dConfig(
             dtype=ttnn.bfloat16,
             weights_dtype=ttnn.bfloat8_b,
-            math_fidelity=ttnn.MathFidelity.LoFi,
             activation=self.activation,
             shard_layout=self.shard_layout,
-            math_approx_mode_enabled=True,
-            fp32_dest_acc_enabled=False,
-            act_block_w_div=1,
-            packer_l1_accum_enabled=False,
             input_channels_alignment=16 if self.input_params[3] < 16 else 32,
             transpose_shards=False,
             reshard_if_not_optimal=self.reshard,
             deallocate_activation=self.deallocate,
             reallocate_halo_output=False,
         )
-        if self.act_block_h is not None:
-            conv_config.act_block_h_override = self.act_block_h
+        compute_config = ttnn.GetComputeKernelConfig(
```

Comment:
why did you change all config values?

Reply:
I had copy-pasted the code snippet. I've fixed the config to be the same as the original.